pkg/test/resourcefixture/testdata/basic/eventarc/v1alpha1/eventarcgooglechannelconfig/eventarcgooglechannelconfig-minimal/update.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: eventarc.cnrm.cloud.google.com/v1alpha1
```

Comment:
I don't see any field being updated in this file. Maybe just remove this file if not used?

Reply:
This resource only has update. Create is the optional one. But the test did not execute without a create. So update is looking like a no-op. Do you know how to deal with update only resource in a test ?

Reply:
Could you try remove the update.yaml and see what happens? I think the test should still work.

Reply:
done. works. 

hack/compare-mock fixtures/eventarcgooglechannelconfig-minimal    shows no diff.